src/groundStation/system.py

```diff
@@ -76,6 +76,61 @@ def __init__(self):
             'LAST': 31
         }
         self.SERVICES = {
+            'SCHEDULER': {
+                'port': 15,  # share a port with EPS time service
+                # TODO: these need a error response value
+                'subservice': {
+                    'SET_SCHEDULE': {
+                        'subPort': 0,
+                        'inoutInfo': {
+                            'args': None,  # All scheduled commands should be stored in schedule.txt
```

Comment:
This is no longer true after adding the argparse stuff

Reply:
@yunqingyi

Reply:
the argparse stuff only selects which text file to upload, it doesn't get passed as a parameter to the subservice